charts/values.mla.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mlaSecrets:
+  minio:
+    ## Set enabled to false if you want to re-use an existing secret for minio.
+    enabled: true
+    ## Set default accesskey, secretkey
+    ## AccessKey and secretKey is generated when not set
+    accessKey: "accesskey"
+    secretKey: "secretkey"
+  grafana:
+    ## Set enabled to false if you want to re-use an existing secret for grafana.
+    enabled: true
+    adminUser: admin
+    adminPassword: strongpassword
+
+iap:
```

Comment:
What is this `iap` section used for? There doesn't seem to be an IAP chart in the MLA directory and the IAP was already installed with the KKP master stack.

Reply:
https://docs.kubermatic.com/kubermatic/master/tutorials-howtos/monitoring-logging-alerting/user-cluster/admin-guide/#expose-grafana--alertmanager-ui

MLA installation instructions suggest installing second instance of IAP together with MLA since MLA is installed per seed so I passed it on here. Do you think we could just reuse the IAP installation on the master in MLA? if so, then it could improve quality of life in general ;-)

Reply:
Yeah when I scrolled further down in the PR, I noticed the other IAP related stuff. It's probably fine. :grin: